src/pages/MyAdvertPage.jsx

```diff
@@ -0,0 +1,9 @@
+const MyAdvertPage = () => {
+    return (
+        <div style={{ marginTop: '50px', fontSize: '50px', textAlign: 'center' }}>
```

Comment:
Please avoid to use inline styles and use styled-components instead

Reply:
This piece of code is temporary and will be removed. So use this approach.